packages/core/deploy/036_deploy_across_ovm_bridge_deposit_box.js

```diff
@@ -0,0 +1,30 @@
+// This deploy script should be run on an optimism provider.
```

Comment:
This script follows a bit of a different pattern to what we've done before: it checks the `chainID` against a set of known IDs and uses this the find the associated `BridgeAdmin` address on L1.